spec/models/comfy/cms/page_spec.rb

```diff
@@ -1,5 +1,57 @@
 RSpec.describe Comfy::Cms::Page do
 
+  describe 'suppress_from_links_recirculation' do
+    let(:english_site) { create :site, is_mirrored: true }
+    let(:welsh_site) { create :site, :welsh, is_mirrored: true }
+
+    let!(:english_article) do
+      create :english_article,
+             full_path:  '/animals',
+             site: english_site
+    end
+    let!(:welsh_article) do
+      create :welsh_article,
+             full_path:  '/animals',
+             site: welsh_site
+    end
+
+    context 'update welsh article when updating english_article' do
+      it 'to be suppressed' do
+        english_article.suppress_from_links_recirculation = true
+        english_article.suppress_mirrors_from_links_recirculation
+        welsh_article.reload
+
+        expect(welsh_article.suppress_from_links_recirculation).to be_truthy
```

Comment:
Yeah, that's what I meant. When calling the getter or setter,
`page.suppress_from_links_recirculation = true/false`
and
`page.suppress_from_links_recirculation?`
`suppress` makes sense.

If we flip that to,
`page.suppressed_from_links_recirculation = true/false`
and
`page.suppressed_from_links_recirculation?`

It reads like (no matter true/false) it has already been suppress'ed' given the past tense.
Especially for the setter.